generic-template/runtime/src/configs/xcm_config.rs

```diff
@@ -22,10 +22,13 @@ use xcm_builder::{
 };
 use xcm_executor::XcmExecutor;
 
```

Comment:
Initially I agreed, but seems like I cannot do that since we also have a dependency that is called `xcm`. We will rename either one of these, so I'll keep `xcm_config` as is. It's no big deal anyways